test/e2e/proxy_test.go

```diff
@@ -76,6 +78,28 @@ func testEnvVars(t *testing.T) {
 			}
 		})
 	}
+	var patches []*patch.JSONPatch
+	patches = append(patches, patch.NewJSONPatch("remove", "/spec/httpProxy", "httpProxy"),
+		patch.NewJSONPatch("remove", "/spec/httpsProxy", "httpsProxy"))
+	patchData, err := json.Marshal(patches)
+	require.NoErrorf(t, err, "%v", patches)
+	_, err = tc.client.Config.ConfigV1().Proxies().Patch(
+		context.TODO(),
+		"cluster",
+		types.JSONPatchType,
+		patchData,
+		meta.PatchOptions{},
+	)
+	patchString := string(patchData)
+	require.NoErrorf(t, err, "unable to patch %s", patchString)
+	time.Sleep(5 * time.Minute)
```

Comment:
please comment why this sleep is needed. Can we call `waitForConfiguredWindowsNodes()` or something similar instead?